draft/ZEP0002.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: default
+title: ZEP0002
+description: This ZEP proposes to add a sharding storage transformer to the
+  Zarr specification version 3. This allows to combine multiple chunks into
+  storage keys, enabling the usage of larger arrays with Zarr.
+parent: draft ZEPs
+nav_order: 1
```

Comment:
The `nav_order` should be 2 as 1 is assigned to [ZEP0001](https://zarr.dev/zeps/draft/ZEP0001.html).